lms/static/js/ccx/schedule.js

```diff
@@ -331,6 +356,10 @@ var edx = edx || {};
 	      });
 	},
 
+	disabled_fields: function($selector, disabled) {
```

Comment:
This name is very confusing. `disabled_field()` sounds a function that returns the fields that are disabled. In this case, you're using this function to disable and enable fields. I suggest that you create two functions, one called `disable_fields()` and one called `enable_fields()`, where the former disables fields and the latter enables them.